src/components/calcite-radio-button-group/calcite-radio-button-group.tsx

```diff
@@ -0,0 +1,65 @@
+import { Component, Host, h, Element, Prop, Listen } from "@stencil/core";
+import { Scale } from "../../interfaces/common";
+
+@Component({
+  tag: "calcite-radio-button-group",
+  styleUrl: "calcite-radio-button-group.scss",
+  shadow: true,
+})
+export class CalciteRadioButtonGroup {
+  @Element() el!: HTMLElement;
+
+  @Prop() disabled: boolean = false;
+  @Prop() name: string;
+  @Prop() scale: Scale = "m";
+  @Prop({ reflect: true }) vertical: boolean = false;
```

Comment:
Nitpick - might be nice to match nomenclature here: https://github.com/Esri/calcite-components/blob/30d7338f75a997f3fba4d6d46fb5260a86c8e782/src/components/calcite-stepper/calcite-stepper.tsx#L49 https://github.com/Esri/calcite-components/blob/30d7338f75a997f3fba4d6d46fb5260a86c8e782/src/components/calcite-stepper/calcite-stepper.tsx#L76